chromium/storage.js

```diff
@@ -1,7 +1,7 @@
-var storage = chrome.storage.local;
+const storage = chrome.storage.local;
 if (chrome.storage.sync) {
   storage = chrome.storage.sync;
```

Comment:
@koops76 Do you ever build and tested it?
https://www.dropbox.com/s/0g4i050j2ivdwv3/https-everywhere-2017.7.5~pre.crx?dl=0

Errors in Chrome 59:
```
storage.js:3 Uncaught TypeError: Assignment to constant variable.
    at storage.js:3
util.js:1 Uncaught SyntaxError: Identifier 'VERB' has already been declared
    at util.js:1
rules.js:220 Uncaught TypeError: Cannot use 'in' operator to search for 'www.miit.gov.cn' in undefined
    at RuleSets.addUserRule (rules.js:220)
    at loadStoredUserRules (background.js:93)
    at background.js:98
background.js:11 [Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.
loadExtensionFile @ background.js:11
_generated_background_page.html:1 Error in response to tabs.query: ReferenceError: activeRulesets is not defined
    at Object.chrome.tabs.query.tabs [as callback] (chrome-extension://pdnbogcodofhjeppkgbjpdmjbjcpbdfd/background.js:112:36)
    at updateState (chrome-extension://pdnbogcodofhjeppkgbjpdmjbjcpbdfd/background.js:108:15)
    at Object.storage.get.item [as callback] (chrome-extension://pdnbogcodofhjeppkgbjpdmjbjcpbdfd/background.js:52:3)
_generated_background_page.html:1 Error in response to tabs.query: ReferenceError: activeRulesets is not defined
    at Object.chrome.tabs.query.tabs [as callback] (chrome-extension://pdnbogcodofhjeppkgbjpdmjbjcpbdfd/background.js:112:36)
    at updateState (chrome-extension://pdnbogcodofhjeppkgbjpdmjbjcpbdfd/background.js:108:15)
    at updateEnabledDisabledUI (chrome-extension://pdnbogcodofhjeppkgbjpdmjbjcpbdfd/popup.js:109:18)
    at HTMLDocument.<anonymous> (chrome-extension://pdnbogcodofhjeppkgbjpdmjbjcpbdfd/popup.js:159:3)
2_generated_background_page.html:1 Error in response to tabs.query: ReferenceError: activeRulesets is not defined
    at Object.chrome.tabs.query.tabs [as callback] (chrome-extension://pdnbogcodofhjeppkgbjpdmjbjcpbdfd/background.js:112:36)
    at updateState (chrome-extension://pdnbogcodofhjeppkgbjpdmjbjcpbdfd/background.js:108:15)
    at chrome.windows.onFocusChanged.addListener (chrome-extension://pdnbogcodofhjeppkgbjpdmjbjcpbdfd/background.js:68:3)
```

an piece errors in Firefox (Nightly) 56a1:
```
TypeError: invalid assignment to const `storage' storage.js:3:3
SyntaxError: redeclaration of const VERB  util.js:1:1
invalid 'in' operand this.active_tab_rules  background.js:197
invalid 'in' operand this.active_tab_rules  background.js:197
```